docs/schema/v0/flag_manifest.json

```diff
@@ -0,0 +1,122 @@
+                    "$ref": "#/$defs/boolType"
+                },
+                {
+                    "$ref": "#/$defs/stringType"
+                },
+                {
+                    "$ref": "#/$defs/intType"
+                },
+                {
+                    "$ref": "#/$defs/floatType"
+                },
+                {
+                    "$ref": "#/$defs/objectType"
+                }
+            ],
+            "required": ["flag_type", "default_value"]
+        },
+        "boolType": {
+            "type": "object",
+            "properties": {
+                "flag_type": {
+                    "type": "string",
+                    "enum": ["bool"]
+                },
+                "default_value": {
+                    "type": "boolean" 
+                },
+                "description": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        },
+        "stringType": {
+            "type": "object",
+            "properties": {
+                "flag_type": {
+                    "type": "string",
+                    "enum": ["string"]
+                },
+                "default_value": {
+                    "type": "string" 
+                },
+                "description": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        },
+        "intType": {
+            "type": "object",
+            "properties": {
+                "flag_type": {
+                    "type": "string",
+                    "enum": ["int"]
+                },
+                "default_value": {
+                    "type": "integer" 
+                },
+                "description": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        },
+        "floatType": {
+            "type": "object",
+            "properties": {
+                "flag_type": {
+                    "type": "string",
+                    "enum": ["float"]
+                },
+                "default_value": {
+                    "type": "number" 
+                },
+                "description": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        },
+        "objectType": {
+            "type": "object",
+            "properties": {
+                "flag_type": {
+                    "type": "string",
+                    "enum": ["object"]
+                },
+                "default_value": {
+                    "type": "object" 
+                },
+                "description": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        }    
+    },
+    "title": "Flag Manifest",
+    "description": "Manifest meant to describe the flags of one flag configuration",
```

Comment:
I'm marking this because I think we should improve this description. However, I don't have a recommendation at the moment, so I wanted to start a conversation.

Reply:
@moredip, you have a way with words 😄 . Do you have a suggestion?

Reply:
How about "Describes a configuration of OpenFeature flags, including info such as their types and default values." (based on what I've seen in flagd.